third_party/repositories/scala_2_11.bzl

```diff
@@ -0,0 +1,416 @@
+artifacts = {
```

Comment:
Awesome!! How did you create this list? Did you use some kind of generator?
What happens when the next dev wants to update something?

Reply:
Currently it's a manual procedure to update deps